### ISSUE TITLE
path.positionA not always on path when using the exact method

### ISSUE THREAD

**Issue by nicolas-UKLSL** [pbrod/nvector#11]
Hello,

First of all, thank you for making this package opensource. It is very useful to me.
I came across the following issue:
on_path when method='exact' will not return consistently True on the positionA of a path.
Below is an illustrating example. Having had a quick look at the code under the hood, it may be the checking the difference of azimuth is not meaningful when `point_a==point`

```
import nvector as nv
import numpy as np

wgs84 = nv.FrameE(name='WGS84')
pointA = wgs84.GeoPoint(latitude=81, longitude=0, degrees=True)
pointB = wgs84.GeoPoint(latitude=80, longitude=0, degrees=True)
path = nv.GeoPath(pointA, pointB)
print(np.allclose(path.on_path(path.positionA, method='exact'), True))

wgs84 = nv.FrameE(name='WGS84')
pointA = wgs84.GeoPoint(latitude=79, longitude=0, degrees=True)
pointB = wgs84.GeoPoint(latitude=80, longitude=0, degrees=True)
path = nv.GeoPath(pointA, pointB)
print(np.allclose(path.on_path(path.positionA, method='exact'), True))
```

nvector.__version__ == 0.7.4

**Comment by pbrod** [comment]
Thank you for reporting this issue.
You are absolutely right. If point_a == point the azimuth can be anything and therefore should not be checked.
I will fix this soon.

**Comment by pbrod** [comment]
The issue is fixed in commit [f9bfc79](https://github.com/pbrod/nvector/commit/f9bfc7918e9f3079ab0eed0704a95e4e55c14854)

**Comment by nicolas-UKLSL** [comment]
Thank you
